src/test/java/org/folio/test/support/AccountsActionAPITests.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.test.support;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+import io.restassured.response.Response;
+import java.util.Date;
+import org.folio.rest.jaxrs.model.Account;
+import org.folio.rest.utils.ResourceClient;
+
+public abstract class AccountsActionAPITests extends ApiTests {
+
+  public static Account verifyAccountAndGet(
+    ResourceClient accountsClient,
+    String accountId,
+    String expectedPaymentStatus,
+    float amount,
```

Comment:
Can we accept `amount` as a `double` and convert it to `float` right before comparison? Just so we could test values retrieved directly from JSONs without casting them every time.